src/log.js

```diff
@@ -255,10 +261,33 @@ class Log extends GSet {
     const newTime = Math.max(this.clock.time, this.heads.reduce(maxClockTimeReducer, 0)) + 1
     this._clock = new Clock(this.clock.id, newTime)
 
-    const references = this.traverse(this.heads, Math.max(pointerCount, this.heads.length))
+    const all = Object.values(this.traverse(this.heads, pointerCount))
```

Comment:
I think we need to increase the default pointerCount to 32 here (few lines above) https://github.com/orbitdb/ipfs-log/pull/275/files#diff-4f24bb7c46eda8f63690cb849ddb0f2dR259. 

This is because with the log optimizations, we would now add 0 refs and only the nexts, which defeats the purpose of the optimizations (with 1 next/ref, it's basically fully sequential and no loading happens in parallel). Also, currently, we save 32 nexts which gives us the "load 32 at a time". 

What do you think @shamb0t?

Reply:
True, the log wont benefit from optimization by default here, 32 sounds good

Reply:
I mixed up ipfs-log and orbit-db :slightly_smiling_face: we've "always" had next ref count be 1 in ipfs-log whereas in orbit-db the default has been 32/64.

I reckon we can leave this as it is now?